### DISCUSSION TITLE
Active Directory not working, no logging

### DISCUSSION THREAD

**Discussion by lestd** [netbox-community/netbox-docker#1119] [Getting Help]
Sorry, another case of AD issues. Been hacking at this with no success, after reviewing other discussions. Even after enabling DEBUG, not getting any errors to work from. Any help is most appreciated.

### System info:
[root@netbox netbox-docker]# uname -a
Linux coripamprdx01 5.14.0-362.8.1.el9_3.x86_64 #1 SMP PREEMPT_DYNAMIC Tue Oct 3 11:12:36 EDT 2023 x86_64 x86_64 x86_64 GNU/Linux
[root@netbox netbox-docker]# docker --version
Docker version 24.0.7, build afdd53b
[root@netbox netbox-docker]# docker-compose --version
Docker Compose version v2.18.0
[root@netbox netbox-docker]# docker image ls
REPOSITORY               TAG          IMAGE ID       CREATED       SIZE
netboxcommunity/netbox   v3.6-2.7.0   c4771298390b   3 weeks ago   688MB
redis                    7-alpine     246a9110fd9e   3 weeks ago   43.4MB
postgres                 15-alpine    2b35f9941b94   3 weeks ago   242MB
caddy                    2-alpine     70c3913f54e2   6 weeks ago   49.1MB

### Configs:
[root@netbox netbox-docker]# cat docker-compose.override.yml

version: '3.4'
services:
  netbox:
    ports:
      - "8000:8080"
    environment:
      REMOTE_AUTH_ENABLED: "True"
      REMOTE_AUTH_BACKEND: "netbox.authentication.LDAPBackend"
      AUTH_LDAP_SERVER_URI: "ldaps://redacted.com"
      AUTH_LDAP_BIND_AS_AUTHENTICATING_USER: "True"
      AUTH_LDAP_BIND_DN: "CN=netboxbind,OU=Netbox,OU=All Security Groups,DC=redacted,dc=com"
      AUTH_LDAP_BIND_PASSWORD: "redacted"
      AUTH_LDAP_USER_SEARCH_BASEDN: "OU=All Users,DC=redacted,dc=com"
      AUTH_LDAP_GROUP_SEARCH_BASEDN: "OU=All Security Groups,DC=redacted,dc=com"
      AUTH_LDAP_REQUIRE_GROUP_DN: "CN=netbox_users,OU=Netbox,OU=All Security Groups,DC=redacted,dc=com"
      AUTH_LDAP_GROUP_TYPE: "ActiveDirectoryGroupType"
      AUTH_LDAP_ALWAYS_UPDATE_USER: "True"
      AUTH_LDAP_IS_ADMIN_DN: "CN=netbox_admins,OU=Netbox,OU=All Security Groups,DC=redacted,dc=com"
      AUTH_LDAP_IS_SUPERUSER_DN: "CN=netbox_super,OU=Netbox,OU=All Security Groups,DC=redacted,dc=com"
      LDAP_IGNORE_CERT_ERRORS: "false"
      LOGLEVEL: "DEBUG"

  tls:
    image: caddy:2-alpine
    depends_on:
      - netbox
    volumes:
      - ./cert.crt:/etc/ssl/private/cert.crt:ro,z
      - ./key.key:/etc/ssl/private/key.key:ro,z
      - ./Caddyfile:/etc/caddy/Caddyfile:ro
    ports:
      - 80:80   # Allows for http redirection
      - 443:443

### logs:
[root@netbox netbox-docker]# docker-compose logs -f netbox
netbox-docker-netbox-1  | ↩️ Skip creating the superuser
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/configuration.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/extra.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/ldap_config.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/logging.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/plugins.py'
netbox-docker-netbox-1  | ✅ Initialisation is done.
netbox-docker-netbox-1  | ⏳ Waiting for control socket to be created... (1/10)
netbox-docker-netbox-1  | 2023/12/05 16:50:43 [warn] 6#6 Unit is running unprivileged, then it cannot use arbitrary user and group.
netbox-docker-netbox-1  | 2023/12/05 16:50:43 [info] 6#6 unit 1.30.0 started
netbox-docker-netbox-1  | 2023/12/05 16:50:43 [info] 12#12 discovery started
netbox-docker-netbox-1  | 2023/12/05 16:50:43 [notice] 12#12 module: python 3.11.2 "/usr/lib/unit/modules/python3.11.unit.so"
netbox-docker-netbox-1  | 2023/12/05 16:50:43 [info] 6#6 controller started
netbox-docker-netbox-1  | 2023/12/05 16:50:43 [notice] 6#6 process 12 exited with code 0
netbox-docker-netbox-1  | 2023/12/05 16:50:43 [info] 14#14 router started
netbox-docker-netbox-1  | 2023/12/05 16:50:43 [info] 14#14 OpenSSL 3.0.8 7 Feb 2023, 30000080
netbox-docker-netbox-1  | ⚙️ Applying configuration from /etc/unit/nginx-unit.json
netbox-docker-netbox-1  | 2023/12/05 16:50:44 [info] 17#17 "netbox" prototype started
netbox-docker-netbox-1  | 2023/12/05 16:50:44 [info] 18#18 "netbox" application started
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/configuration.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/extra.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/ldap_config.py'       <----- loaded
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/logging.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/plugins.py'
netbox-docker-netbox-1  | ✅ Unit configuration loaded successfully
netbox-docker-netbox-1  | 2023/12/05 16:50:45 [notice] 6#6 process 10 exited with code 0
netbox-docker-netbox-1  | 2023/12/05 16:50:49 [info] 29#29 "netbox" application started
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/configuration.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/extra.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/ldap_config.py'           <---- why is this done twice?
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/logging.py'
netbox-docker-netbox-1  | 🧬 loaded config '/etc/netbox/config/plugins.py'
netbox-docker-netbox-1  | 127.0.0.1 - - [05/Dec/2023:16:50:51 +0000] "GET /api/ HTTP/1.1" 200 510 "-" "curl/7.88.1"

**Comment by Hacyran** [comment]
Hello.

Same here. Added environment lines to docker-compose.override.yml, set DEBUG=True *and* LOGLEVEL=DEBUG (just in case), removed comment prefix from configuration/logging.py, logged in netbox-docker-netbox-1 and verified the vars are set. I get the "Debug mode is enabled" message in netbox webui, yet nothing relevant from "docker-compose logs -f netbox netbox-worker".

Did you get any improvement on this ?
